rainier-core/src/main/scala/com/stripe/rainier/repl/package.scala

```diff
@@ -194,7 +195,7 @@ package object repl {
       models,
       Sampler.Default.sampler,
       Sampler.Default.warmupIterations,
-      Sampler.Default.iterations,
+      Sampler.Default.iterations
     )
 
   def compare(models: Seq[(String, RandomVariable[_])],
```

Comment:
I think this form of `compare` should be replaced by one that takes a `Seq[(String, Sample[_])]`. Also, should the models in this form be a vararg to match the other form?

Reply:
I agree that we should have a form of `compare` that takes `Sample` but I think I'd like to defer this until a more comprehensive refactoring.